charts/openmetadata/templates/serviceaccount.yaml

```diff
@@ -4,9 +4,9 @@ kind: ServiceAccount
 metadata:
   name: {{ include "OpenMetadata.serviceAccountName" . }}
   labels:
-    {{- include "OpenMetadata.labels" . | nindent 4 }}
-  {{- with .Values.serviceAccount.annotations }}
+    {{- include "OpenMetadata.labels" . | nindent 4 }} 
   annotations:
+    {{- with .Values.serviceAccount.annotations }}
```

Comment:
All the Kubernetes Objects should refer to this by calling
{{- include "OpenMetadata.annotations" . | nindent 4 }}.

Refactor to _helper.tpl as mentioned above in review.

Reply:
added {{- include "OpenMetadata.annotations" . | nindent 4 }}